client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -416,7 +416,7 @@ export const SeoForm = React.createClass( {
 		const activateVerificationServices = () => this.props.activateModule( siteId, 'verification-tools' );
 		const isSitePrivate = siteSettings && parseInt( siteSettings.blog_public, 10 ) !== 1;
 		const isJetpackUnsupported = siteIsJetpack && ! jetpackVersionSupportsSeo;
-		const isDisabled = isSitePrivate || isJetpackUnsupported || isSubmittingForm || isFetchingSettings;
+		const isDisabled = isJetpackUnsupported || isSubmittingForm || isFetchingSettings;
```

Comment:
Cool, I'll be happy to give it another shot after you update it to work for `hidden`.